python/semantic_kernel/contents/chat_history.py

```diff
@@ -193,7 +193,7 @@ def from_rendered_prompt(
         try:
             xml_prompt = ET.fromstring(f"<prompt>{prompt}</prompt>")
         except ET.ParseError as e:
-            logger.error(f"Error parsing XML of prompt: {e}")
+            logger.warning(f"Unable to parse XML from prompt: {e}")
```

Comment:
I changed this to be a warning, but I think we should remove it all together. It's a bit misleading -- this code is hit because the planner's prompt isn't formed to pass this parsing, yet everything still works in the end. It was misleading to me, as I thought the step to create the plan errored out, but it worked in the end.